lib/libvarnish/vcli_serve.c

```diff
@@ -102,6 +102,21 @@ VCLS_func_ping(struct cli *cli, const char * const *av, void *priv)
 	VCLI_Out(cli, "PONG %jd 1.0", (intmax_t)t);
 }
 
+void v_matchproto_(cli_func_t)
+VCLS_func_ping_json(struct cli *cli, const char * const *av, void *priv)
+{
+	time_t t;
+
+	(void)av;
+	(void)priv;
+	t = time(NULL);
+	VCLI_JSON_begin(cli, 1, av);
+	VSB_indent(cli->sb, 2);
+	VCLI_Out(cli, ",\n {\n \"PONG\": %jd \n }", (intmax_t)t);
+	VSB_indent(cli->sb, -2);
```

Comment:
These VSB_indent calls don't really follow the output from `help -j`, which if I'm reading right should be:

```
  {
     "PONG": 1234
  }
```

Reply:
Thanks! you're absolutely right. I've pushed a new version following the same format as in cache_director.c, lines [373-381].